sound/soc/sof/sof-priv.h

```diff
@@ -208,6 +209,12 @@ struct snd_sof_dsp_ops {
 	int (*pre_fw_run)(struct snd_sof_dev *sof_dev); /* optional */
 	int (*post_fw_run)(struct snd_sof_dev *sof_dev); /* optional */
 
+	/* parse platform specific extended manifest */
+	int (*parse_platform_ext_manifest)(struct snd_sof_dev *sof_dev,
+					   const struct sof_ext_man_elem_header *hdr); /* optional */
+	/* enable last core for iccmax support */
+	void (*iccmax_enable_last_core)(struct snd_sof_dev *sof_dev); /* optional */
```

Comment:
As commented elsewhere already, let's use existing ops to enable the additional core for iccmax.